content/en/docs/concepts/package-format.md

```diff
@@ -47,22 +47,6 @@ Well-Known Annotations are used to control the loading behavior of an object wit
 ```yaml
 apiVersion: manifests.package-operator.run/v1alpha1
 kind: PackageManifest
-catalog:
```

Comment:
Pardon my ignorance, why is this removed?

Reply:
Because it's not a thing ;)